### ISSUE TITLE
reminders (sms, email, todoist)

### ISSUE THREAD

**Issue by PeerRich** [calcom/cal.com#1780]
ability to create reminders for event types, both
- N days/hours/minutes before the event (reminder)
- N days/hours after the event (follow up)

we could implement todoist too:
- before: [] prepare for {EventName}
- after: [] follow up with {EventName}

add webhook support too @alishaz-polymath

**Comment by PeerRich** [comment]
@Jaibles wanna design a quick mock how this could look like in the event-types detail?

**Comment by agustif** [comment]
Leaving this lib I've looked up in the past that could be useful to integrate different services other than email here 
https://github.com/notifme/notifme-sdk

**Comment by PeerRich** [comment]
> Leaving this lib I've looked up in the past that could be useful to integrate different services other than email here https://github.com/notifme/notifme-sdk

that's super smart!

**Comment by adowbor** [comment]
@PeerRich I think it would be interesting to have a webhook fire when these events are triggered, this way we can connect to any other service as required. You may want to receive a Slack notification for example or send an update to a CRM system.

**Comment by PeerRich** [comment]
> @PeerRich I think it would be interesting to have a webhook fire when these events are triggered, this way we can connect to any other service as required. You may want to receive a Slack notification for example or send an update to a CRM system.

cc @alishaz-polymath

**Comment by joeauyeung** [comment]
Taking myself off in favour of [#2861](https://github.com/calcom/cal.com/issues/2861)

**Comment by PeerRich** [comment]
duplicate of #2861
